src/main/bash/envs/kobman-aries-dev.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+function __kobman_install_aries-dev
+{
+    local environment=$1
+    local version=$2
+    if [[ -z $KOBMAN_ARIES_ENV_ROOT ]]; then
+        export KOBMAN_ARIES_ENV_ROOT=$HOME/Aries_dev
+    fi
+    check_python3_installation
+    check_pip_installation
+    __kobman_echo_white "Installing $environment environment"
+    pip3 install aries-cloudagent
+    if [[ ! -d $KOBMAN_ARIES_ENV_ROOT ]]; then
+		# __kobman_create_fork "${environment}" || return 1
+
+		__kobman_create_aries_dev_environment "$environment" "$version" || return 1
+		
+	else
+ 		__kobman_echo_white "Removing existing version "
+		rm -rf $KOBMAN_ARIES_ENV_ROOT
+		__kobman_create_dev_environment "$environment" "$version" || return 1
+	fi
+}
+
+function __kobman_create_aries_dev_environment 
+{
+	
+	local environment=$1
+    local version=$2
+    __kobman_echo_white "Creating Dev environment for ${environment} under $KOBMAN_ARIES_ENV_ROOT"
+    # __kobman_echo_white "from https://github.com/${KOBMAN_USER_NAMESPACE}/${environment}"
+    __kobman_echo_white "version :${version} "
+	mkdir -p $KOBMAN_ARIES_ENV_ROOT
+	# git clone -q https://github.com/$KOBMAN_USER_NAMESPACE/${environment} $KOBMAN_ARIES_ENV_ROOT/$environment
```

Comment:
@asa1997 , need to be removed along with the nxt pull . ( only if not required )